api/src/main/java/org/cardanofoundation/rosetta/api/block/model/entity/projection/BlockIdentifierProjection.java

```diff
@@ -0,0 +1,8 @@
+package org.cardanofoundation.rosetta.api.block.model.entity.projection;
+
+public interface BlockIdentifierProjection {
```

Comment:
Just FYI, it could be significantly slow:
https://stackoverflow.com/questions/53347063/why-are-interface-projections-much-slower-than-constructor-projections-and-entit

Let's just keep that in mind during the performance results review